storybook/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -5,96 +5,86 @@
 
 import { Heading, Paragraph, Tabs } from '@amsterdam/design-system-react/src'
 import { Meta, StoryObj } from '@storybook/react'
-import { PropsWithChildren } from 'react'
+import { PropsWithChildren, ReactNode } from 'react'
 import { exampleParagraph } from '../shared/exampleContent'
 
+const slowPanelDelay = 1000
+
 const SlowPanel = ({ children }: PropsWithChildren) => {
-  console.log('[ARTIFICIALLY SLOW] Adding a 1000ms delay')
+  console.log(`[ARTIFICIALLY SLOW] Adding a ${slowPanelDelay}ms delay…`)
 
   let startTime = performance.now()
-  while (performance.now() - startTime < 1000) {
-    // Do nothing for 1000 ms to emulate extremely slow code
+  while (performance.now() - startTime < slowPanelDelay) {
+    /* Emulate slow code. */
   }
 
   return children
 }
 
-const meta = {
-  title: 'Components/Containers/Tabs',
-  component: Tabs,
-} satisfies Meta<typeof Tabs>
-
-export default meta
+type TabContent = {
+  label: string
+  body: ReactNode
+}
 
-const tabMeta = {
-  component: Tabs.Button,
-  decorators: [
-    (Story) => (
-      <Tabs>
-        <Story />
-      </Tabs>
+const tabsContent: Array<TabContent> = [
+  { label: 'Gegevens', body: exampleParagraph() },
+  { label: 'Aanslagen', body: exampleParagraph() },
+  {
+    label: 'Documenten',
+    body: (
+      <>
+        (This tab panel simulates a load time of {slowPanelDelay} milliseconds.)
+        <SlowPanel />
+      </>
     ),
-  ],
-} satisfies Meta<typeof Tabs.Button>
-
-type Story = StoryObj<typeof meta>
-type TabStory = StoryObj<typeof tabMeta>
+  },
+  { label: 'Acties', body: exampleParagraph() },
+]
 
-export const Default: Story = {
-  args: {
-    children: [
-      <Tabs.List key={0}>
-        <Tabs.Button tab={0}>Gegevens</Tabs.Button>
-        <Tabs.Button tab={1}>Aanslagen</Tabs.Button>
-        <Tabs.Button tab={2}>Documenten</Tabs.Button>
-        <Tabs.Button tab={3}>Acties</Tabs.Button>
-      </Tabs.List>,
-      <Tabs.Panel tab={0} key={1}>
-        <div style={{ paddingTop: '2rem' }}>
-          <Heading level={3}>Gegevens</Heading>
-          <Paragraph>{exampleParagraph()}</Paragraph>
-        </div>
-      </Tabs.Panel>,
-      <Tabs.Panel tab={1} key={2}>
-        <div style={{ paddingTop: '2rem' }}>
-          <Heading level={3}>Aanslagen</Heading>
-          <Paragraph>{exampleParagraph()}</Paragraph>
-        </div>
-      </Tabs.Panel>,
-      <Tabs.Panel tab={2} key={3}>
+const defaultTabs = [
+  <Tabs.List key="tabsList">
+    {tabsContent.map((content, index) => (
+      <Tabs.Button key={content.label} tab={index}>
+        {content.label}
+      </Tabs.Button>
+    ))}
+  </Tabs.List>,
+  ...(() =>
```

Comment:
Curious why an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) is better here? What’s the advantage over ‘just’ looping over `tabsContent`?

Reply:
Yeah, I don't know why I did this. I think we can just put the map function in parentheses as well.

Reply:
I’ve made it so.